pkg/client/unversioned/client.go

```diff
@@ -149,7 +149,7 @@ func (c *Client) ServerVersion() (*version.Info, error) {
 
 // ServerAPIVersions retrieves and parses the list of API versions the server supports.
 func (c *Client) ServerAPIVersions() (*unversioned.APIVersions, error) {
```

Comment:
Thanks @krousey. This reminds me we should remove ServerAPIVersions() from typed clients since we have the discovery client now.